iceoryx2-ffi/ffi/src/node.rs

```diff
@@ -57,57 +50,88 @@ impl IntoCInt for NodeListFailure {
 }
 
 #[repr(C)]
+#[repr(align(8))] // magic number; the larger one of align_of::<Node<zero_copy::Service>>() and align_of::<Node<process_local::Service>>()
```

Comment:
Just a remark, but you could also write `the maximum of`, which would be clearer.